apps/vanilla/src/pages/2/logick/publisher.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsdoc/require-jsdoc */
+
+import { Subscriber } from './types';
+
+export class Publisher<T> {
+	private readonly subscribers: Subscriber<T>[] = [];
```

Comment:
We also can add `readonly` for array, not only for property.
```suggestion
	private readonly subscribers: readonly Subscriber<T>[] = [];
```